pkg/graph/resolvers/system_intake_actions_test.go

```diff
@@ -0,0 +1,248 @@
+				var stepState models.SystemIntakeFormState
+				switch step {
+				case model.SystemIntakeFormStepInitialRequestForm:
+					stepState = actionedIntake.RequestFormState
+				case model.SystemIntakeFormStepDraftBusinessCase:
+					stepState = actionedIntake.DraftBusinessCaseState
+				case model.SystemIntakeFormStepFinalBusinessCase:
+					stepState = actionedIntake.FinalBusinessCaseState
+				default:
+					stepState = ""
+				}
+				s.Equal(models.SIRFSEditsRequested, stepState)
+				// test that step is changed to requested step
+				s.Equal(formStepMap[step], actionedIntake.Step)
+			})
+		}
+	}
+	// test that feedback is created
+	s.Run("Should create feedback", func() {
+		intake, err := s.testConfigs.Store.CreateSystemIntake(ctx, &models.SystemIntake{
+			Status:      models.SystemIntakeStatusINTAKEDRAFT,
+			RequestType: models.SystemIntakeRequestTypeNEW,
+			Step:        models.SystemIntakeStepINITIALFORM,
+		})
+		s.NoError(err)
+		addNotes := "banana"
+		adminNotes := "apple"
+		actionedIntake, err := CreateSystemIntakeActionRequestEdits(
+			ctx,
+			s.testConfigs.Store,
+			s.fetchUserInfoStub,
+			model.SystemIntakeRequestEditsInput{
+				SystemIntakeID: intake.ID,
+				IntakeFormStep: model.SystemIntakeFormStepInitialRequestForm,
+				NotificationRecipients: &models.EmailNotificationRecipients{
+					RegularRecipientEmails:   []models.EmailAddress{"banana"},
+					ShouldNotifyITGovernance: false,
+					ShouldNotifyITInvestment: false,
+				},
+				EmailFeedback:   "meatloaf",
+				AdditionalNotes: &addNotes,
+				AdminNotes:      &adminNotes,
+			},
+		)
+		s.NoError(err)
+		s.Equal(intake.ID, actionedIntake.ID)
+		allFeedback, err := s.testConfigs.Store.GetGovernanceRequestFeedbacksByIntakeID(ctx, actionedIntake.ID)
+		s.NoError(err)
+		createdFeedback := allFeedback[0]
+		s.Equal("meatloaf", createdFeedback.Feedback)
+	})
+	s.Run("Should create action", func() {
+		intake, err := s.testConfigs.Store.CreateSystemIntake(ctx, &models.SystemIntake{
+			Status:      models.SystemIntakeStatusINTAKEDRAFT,
+			RequestType: models.SystemIntakeRequestTypeNEW,
+			Step:        models.SystemIntakeStepINITIALFORM,
+		})
+		s.NoError(err)
+		addNotes := "banana"
+		adminNotes := "apple"
+		actionedIntake, err := CreateSystemIntakeActionRequestEdits(
+			ctx,
+			s.testConfigs.Store,
+			s.fetchUserInfoStub,
+			model.SystemIntakeRequestEditsInput{
+				SystemIntakeID: intake.ID,
+				IntakeFormStep: model.SystemIntakeFormStepInitialRequestForm,
+				NotificationRecipients: &models.EmailNotificationRecipients{
+					RegularRecipientEmails:   []models.EmailAddress{"banana"},
+					ShouldNotifyITGovernance: false,
+					ShouldNotifyITInvestment: false,
+				},
+				EmailFeedback:   "meatloaf",
+				AdditionalNotes: &addNotes,
+				AdminNotes:      &adminNotes,
+			},
+		)
+		s.NoError(err)
+		s.Equal(intake.ID, actionedIntake.ID)
+		allActions, err := s.testConfigs.Store.GetActionsByRequestID(ctx, actionedIntake.ID)
```

Comment:
It's probably good to assert the new step field has been updated as expected.

Reply:
Addressed in [4aa9736](https://github.com/CMSgov/easi-app/pull/2129/commits/4aa97368d9b0789d99741ee6968dadc88b103064)